sso-kit-keycloak-theme/src/main/resources/theme/sso-kit-keycloak-theme/login/template.ftl

```diff
@@ -0,0 +1,180 @@
+<#macro registrationLayout bodyClass="" displayInfo=false displayMessage=true displayRequiredFields=false>
+    <!DOCTYPE html>
+    <html class="${properties.kcHtmlClass!}">
+
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+        <meta name="robots" content="noindex, nofollow">
+
+        <#if properties.meta?has_content>
+            <#list properties.meta?split(' ') as meta>
+                <meta name="${meta?split('==')[0]}"
+                      content="${meta?split('==')[1]}"/>
+            </#list>
+        </#if>
+        <title>${msg("loginTitle",(realm.displayName!''))}</title>
+        <link rel="icon" href="${url.resourcesPath}/img/favicon.ico"/>
+        <#if properties.stylesCommon?has_content>
+            <#list properties.stylesCommon?split(' ') as style>
+                <link href="${url.resourcesCommonPath}/${style}"
+                      rel="stylesheet"/>
+            </#list>
+        </#if>
+        <#if properties.styles?has_content>
+            <#list properties.styles?split(' ') as style>
+                <link href="${url.resourcesPath}/${style}" rel="stylesheet"/>
+            </#list>
+        </#if>
+        <#if properties.scripts?has_content>
+            <#list properties.scripts?split(' ') as script>
+                <script src="${url.resourcesPath}/${script}"
+                        type="text/javascript"></script>
+            </#list>
+        </#if>
+        <#if scripts??>
+            <#list scripts as script>
+                <script src="${script}" type="text/javascript"></script>
+            </#list>
+        </#if>
+    </head>
+
+    <body class="${properties.kcBodyClass!}">
+    <div class="${properties.kcLoginClass!}">
+        <div id="kc-header" class="${properties.kcHeaderClass!}">
+            <div id="kc-header-wrapper"
+                 class="${properties.kcHeaderWrapperClass!}">
+                ${kcSanitize(msg("loginTitleHtml",(realm.displayNameHtml!'')))?no_esc}
+            </div>
```

Comment:
If `realm.displayNameHtml` is not defined, can we have a default `<h1>` with `realm.displayName` here?

Reply:
Done. I noticed that `realm.displayNameHtml` defaults to `realm.displayName` when nothing is provided at the "HTML Display name" field on Keycloak, so the way I figure to check when a custom markup is provided is to test when those two properties are different. When no `realm.displayNameHtml` is provided, the `realm.displayName` is shown inside a `<h1>` tag.